crates/bevy_transform/src/hierarchy/child_builder.rs

```diff
@@ -88,6 +88,37 @@ impl<'a, 'b> ChildBuilder<'a, 'b> {
         self.push_children.parent
     }
 
+    /// Return a mutable reference to the [`Commands`] the `ChildBuilder` was constructed with.
+    ///
+    /// Calling [`spawn`][Commands::spawn] or [`spawn_bundle`][Commands::spawn_bundle] on the
+    /// returned value will **not** insert the parent and children components.
```

Comment:
This seems like a footgun waiting to happen...

Reply:
It's a footgun on the already-present `add_commands` method.